mdbook-course/Cargo.toml

```diff
@@ -8,6 +8,18 @@ publish = false
 repository = "https://github.com/google/comprehensive-rust"
 description = "An mdbook preprocessor for comprehensive-rust."
 
+[[bin]]
+name = "course-schedule"
+path = "src/bin/course-schedule.rs"
+
+[[bin]]
+name = "mdbook-course"
+path = "src/bin/mdbook-course.rs"
+
+[lib]
+name = "mdbook_course"
+path = "src/lib.rs"
+
```

Comment:
This should not be necessary: what you write here is what Cargo deduces by default (as far as I know). I tested it locally and it seems to work fine without:

```suggestion
```